conversions.go

```diff
@@ -131,25 +148,42 @@ func (m Map) parseURLValues(queryMap Map, vals url.Values, key string) {
 				m.parseURLValues(New(v), vals, key+"["+k+"]")
 			}
 		case val.IsMSISlice():
```

Comment:
This case isn't covered by any testcases. Mind adding some? I think you can reuse the `val.IsObjxMapSlice()` ones

Reply:
Due to a previous pull request, this part is no longer necessary. Updated the code and added tests.